superset/assets/javascripts/dashboard/reducers/dashboard.js

```diff
@@ -0,0 +1,115 @@
+import { merge as mergeArray } from 'd3';
+
+import * as actions from '../actions/dashboard';
```

Comment:
I would import all of these individually. also, should this file (similar to actions) be called `dashboardState`?

Reply:
fixed.